src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.PoA.Voting
+{
+    public class VotingDataModel
+    {
+        public string key { get; private set; }
+        public string hash { get; private set; }
+
+        public VotingDataModel(VotingData votingData)
```

Comment:
@MithrilMan  Also need to a private parameterless constructor and add [JsonConstructor] on it.